plugins/modules/infra_join_token.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Infoblox Inc.
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+---
+module: infra_join_token
+short_description: Manage UIJoinToken
+description:
+    - Manage UIJoinToken
+version_added: 2.0.0
+author: Infoblox Inc. (@infobloxopen)
+options:
+    id:
+        description:
+            - ID of the object
+        type: str
+        required: false
+    state:
+        description:
+            - Indicate desired state of the object
+        type: str
+        required: false
+        choices:
+            - present
+            - absent
+        default: present
+    description:
+        description: ""
+        type: str
+    expires_at:
+        description: ""
+        type: str
+    name:
+        description: ""
+        type: str
+    tags:
+        description: ""
+        type: dict
+
+extends_documentation_fragment:
+    - infoblox.bloxone.common
+"""  # noqa: E501
+
+EXAMPLES = r"""
+  - name: Create a UI Join token
```

Comment:
```suggestion
  - name: Create a Join Token
```

same for the below examples as well

Reply:
removed `UI` keyword from all instances except API Name